lib/pos_blockchain.go

```diff
@@ -144,9 +144,16 @@ func (bc *Blockchain) validateAndIndexHeaderPoS(header *MsgDeSoHeader, headerHas
 			header, errors.New("validateAndIndexHeaderPoS: Parent header failed validations"),
 		)
 	}
-
+	utxoView, err := bc.getUtxoViewAtBlockHash(*header.PrevBlockHash)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "validateAndIndexHeaderPoS: Problem getting UtxoView")
+	}
+	snapshotGlobalParams, err := utxoView.GetCurrentSnapshotGlobalParamsEntry()
```

Comment:
Add a comment here to elaborate on why we use the snapshot global params. It's to make the drift timestamp check work for orphan blocks.